spec/features/add_member_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+require 'spec_helper'
+require_relative './helpers/login_helper'
+require 'selenium-webdriver'
+include LogInHelper
+
+
+feature 'Add someone to a team' do
+  scenario 'send the invitation as sender and receive it', :js=>true do
+
+    # sign in
+    log_in('student4346', 'password')
```

Comment:
Hi, you need to take a look on this post: https://piazza.com/class/idkvxa78cn13c9?cid=403.

For this kind of test you cannot assume that "student4346" and assignment "Ethical analysis 2" are in the testing db. When the test starts, there is only the data in seeds.rb in the testing db. However, the "admin" account is defined and you can use admin to create students' account and assignment before you do your test.